cocos/physics-2d/framework/physics-system.ts

```diff
@@ -13,6 +13,9 @@ import { IPhysicsConfig, ICollisionMatrix } from '../../physics/framework/physic
 import { CollisionMatrix } from '../../physics/framework/collision-matrix';
 import { ERaycast2DType, RaycastResult2D, PHYSICS_2D_PTM_RATIO, PhysicsGroup } from './physics-types';
 import { Collider2D } from './components/colliders/collider-2d';
+import { legacyCC } from '../../core/global-exports';
+
+legacyCC.internal.PhysicsGroup = PhysicsGroup;
```

Comment:
Override will lead to undefined behavior, so we need to sort this problem out with other approach. I will talk to editor team

Reply:
yes, agree! last night i found there were empty dropdown lists for RigidBody2D's `group` member, but user custom `PhysicsGroup` works fine.
so i wanted to comment out "Please don't merge, i found something strange and i need to test it thorough etc blahblah", and just found you here, LOL.
thanks, Panda.